src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -0,0 +1,255 @@
+              :showErrors="false"
+              :businessLookup="initialBusinessLookupObject"
+              :BusinessLookupServices="BusinessLookupServices"
+              label="Business Name or Incorporation Number"
+              @setBusiness="setAmalgamatingBusiness($event)"
+            />
+            <v-row
+              class="justify-end mr-0 mt-2"
+            >
+              <v-btn
+                id="app-cancel-btn"
+                large
+                outlined
+                color="primary"
+                @click="addAmalgamatingBusinessCancel()"
+              >
+                <span>Cancel</span>
+              </v-btn>
+            </v-row>
+          </v-col>
+        </v-row>
+      </v-card>
+    </v-expand-transition>
+
+    <!-- Add an Amalgamating Foreign Business button clicked -->
+    <v-expand-transition>
+      <v-card
+        v-if="isAddingAmalgamatingForeignBusiness"
+        flat
+        class="section-container mt-4"
+      >
+        <v-row
+          no-gutters
+        >
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Add a Foreign Business</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="8"
+            class="ml-8"
+          >
+            <span>**TODO**</span>
+          </v-col>
+
+          <!-- extra column is for possible action button -->
+        </v-row>
+      </v-card>
+    </v-expand-transition>
+    <v-row class="mt-4 ml-1">
+      <ul>
+        Amalgamating Businesses: <br><br>
+        <li
+          v-for="(business, index) in amalgamatingBuinesses"
+          :key="index"
+        >
+          <template v-if="business.foundingDate">
+            Legal Name: {{ business.legalName }} <br>
+            Legal Type: {{ business.legalType }} <br>
+            Mailing Address: {{ business.officeAddress.mailingAddress }} <br>
+            State: {{ business.state }} <br>
+            Good Standing: {{ business.goodStanding }} <br>
+          </template>
+          <template v-else>
+            Legal Name: {{ business.name }} <br>
+            Legal Type: {{ business.legalType }} <br>
+            Identifier: {{ business.identifier }} <br>
+            Status: {{ business.status }}
+          </template>
+        </li>
+      </ul>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator'
+import { Action, Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
+import { CommonMixin } from '@/mixins'
+import { BusinessLookupServices, LegalServices } from '@/services'
+import { BusinessLookup } from '@bcrs-shared-components/business-lookup'
+import { BusinessIF, BusinessLookupIF, EmptyBusinessLookup } from '@/interfaces'
+
+@Component({
+  components: {
+    BusinessLookup
+  }
+})
+export default class AmalgamatingBusinesses extends Mixins(CommonMixin) {
+  @Getter(useStore) getAmalgamatingBusinesses!: Array<BusinessIF>
```

Comment:
Not this getter for now but eventually we'll have to use this getter when continuing a draft.